File: 11_03 Terca/Python/hospede.py

nome = input("Escreva o nome do hóspede: ")
apartamento = input("Escolha o tipo de apartamento \n 1 A \n 2 B \n 3 C \n 4 D \n): ")
diarias = int(input("Número de diárias: "))
consumo = float(input("Consumo interno: "))


if apartamento == "A":
    valor_diaria = 150
elif apartamento == "B":
    valor_diaria = 100
elif apartamento == "C":
    valor_diaria = 75
elif apartamento == "D":
    valor_diaria = 50
else:
    print("Tipo de apartamento inválido.")
    exit()  


total_diaria = diarias * valor_diaria


subtotal = total_diaria + consumo


taxa_servico = subtotal * 0.1


total = subtotal + taxa_servico


print(f"\nO nome do hóspede é: {nome}")
print(f"O tipo de apartamento que ele ficou foi: {apartamento}")
print(f"O número de diárias utilizadas foi: {diarias}")
print(f"O valor da diária para esse tipo de apartamento é: {valor_diaria}")
print(f"O consumo interno foi: {consumo}")
print(f"O subtotal foi: {subtotal}")
print(f"O valor da taxa de serviço foi: {taxa_servico}")
print(f"O total gasto foi: {total}")

File: Programação/14_03 Exercicio Contas/hospede.php
<?php
    $nome = $_GET["nome"];
    $apartamento = $_GET["apartamento"];
    $diaria = $_GET["diaria"];
    $consumo = $_GET["consumo"];

    if($apartamento == "Tipo A")
    {   
        $totalDiaria = $diaria * 150;
        $subtotal = $totalDiaria + $consumo;


    }
    elseif($apartamento == "Tipo B")
    {
        $totalDiaria = $diaria * 100;
        $subtotal = $totalDiaria + $consumo;

    }
    elseif($apartamento == "Tipo C")
    {
        $totalDiaria = $diaria * 75;
        $subtotal = $totalDiaria + $consumo;

    }
    elseif($apartamento == "Tipo D")
    {
        $totalDiaria = $diaria * 50;
        $subtotal = $totalDiaria + $consumo;
    }
    $taxaServico = $subtotal*0.1;
    $total = $subtotal + $taxaServico;
    $valorUnitarioDiaria = 150;
    echo "O nome do hospede é : ". $nome . "<br><br>". " O tipo de apartamento que ele ficou foi : ". $apartamento . "<br><br>". " O número de diárias utilizadas foi : ".
    $diaria . "<br><br>". " O valor da diária para esse tipo de apartamento é : ". $valorUnitarioDiaria . "<br><br>". " O consumo interno foi : ". $consumo . "<br><br>"." O subtotal foi : "
    . $subtotal . "<br><br>". " O valor da taxa de serviço foi : ". $taxaServico . "<br><br>". "O total gasto foi : ". $total;